pkg/cmd/cli/restore/logs.go

```diff
@@ -53,3 +54,33 @@ func NewLogsCommand(f client.Factory) *cobra.Command {
 
 	return c
 }
+
+type LogsOptions struct {
+	RestoreName string
+	client      arkclient.Interface
```

Comment:
I typically have 1 blank line between exported and unexported fields.

Reply:
added